packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -429,9 +483,34 @@ class _DraggableScrollableSheetState extends State<DraggableScrollableSheet> {
   @override
   void dispose() {
     _scrollController.dispose();
+    _extent.dispose();
     super.dispose();
   }
 
+  void _replaceExtent() {
```

Comment:
I named this `_replaceExtent` instead of `_updateExtent` so that it wouldn't get confused with `extent.updateExtent`.